readme.md

```diff
@@ -44,68 +46,38 @@ console.log(stdout);
 
 ### Scripts interface
 
-For more information, please see [this page](docs/scripts.md).
+For more information about Execa scripts, please see [this page](docs/scripts.md).
 
 #### Basic
 
 ```js
 import {$} from 'execa';
 
-const {stdout} = await $`echo unicorns`;
-// const {stdout} = await $`echo ${'unicorns'}`;
-// const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
+const unicorns = await $`echo unicorns`;
+const {stdout} = await $`echo ${unicorns}${'!'}}`;
+console.log(stdout);
+//=> 'unicorns!'
 
+const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
 console.log(stdout);
-//=> 'unicorns'
+//=> 'unicorns rainbows'
 ```
 
 #### With options
 
 ```js
 import {$} from 'execa';
 
-await $({stdio: 'inherit'})`echo unicorns`;
-//=> 'unicorns'
-```
-
-#### With pre-defined options
-
-```js
-import {$} from 'execa';
-
 const $$ = $({stdio: 'inherit'});
 
 await $$`echo unicorns`;
 //=> 'unicorns'
-await $$({shell: true})`echo unicorns && echo rainbows`;
-//=> 'unicorns'
-//=> 'rainbows'
-```
-
-#### Synchronous
-
-```js
-import {$} from 'execa';
 
-const {stdout} = $.sync`echo unicorns`;
-console.log(stdout);
+await $$({shell: true})`echo unicorns && echo rainbows`;
```

Comment:
This consolidation is nice, but removes the basic example of using the imported `$` with options e.g. `` await $(options)`command` ``. I'm not sure if this will lead to confusion, but user might think they have to use the `$$ = $(options)` binding mechanism first.

Reply:
That's a good point, let's keep them split. Done :+1: